CMakeLists.txt

```diff
@@ -216,6 +216,15 @@ zephyr_compile_options(${CONFIG_COMPILER_OPT})
 # TODO: This Clang check is broken
 if(CMAKE_C_COMPILER_ID STREQUAL "Clang")
   zephyr_cc_option(
+    #FIXME: need to fix all of those
```

Comment:
Need bugs filed for these, or at least one bug that describes what needs to be done.